pkg/sql/execinfrapb/processors_bulk_io.proto

```diff
@@ -395,7 +395,7 @@ message GenerativeSplitAndScatterSpec {
   repeated roachpb.Span spans = 10 [(gogoproto.nullable) = false];
   repeated jobs.jobspb.RestoreDetails.BackupLocalityInfo backup_locality_info = 11 [(gogoproto.nullable) = false];
   // HighWater is the high watermark of the previous run of restore.
-  optional bytes high_water = 12 [(gogoproto.nullable) = false];
```

Comment:
intentional?

Reply:
Yeah, it was from a backport that was missed that just removed this no-effect nullable=false